.github/workflows/manual-release.yml

```
@@ -32,3 +32,38 @@ jobs:
+        run: |
+          git config --global user.name "github-actions[bot]"
+          git config --global user.email "github-actions[bot]@users.noreply.github.com"
+          
+      - name: Commit and Push if changed
+        run: |
+          git add dist/index.js
+          git commit -m "chore: build distribution" || echo "No changes to commit"
+          git push origin HEAD:main
+          
+      - name: Create Release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.10)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)